### ISSUE TITLE
Config comments do not specify radiation decay accuratly

### ISSUE THREAD

**Issue by SkullOwner** [mekanism/Mekanism#7118]
#### Issue description:
_Not sure if this is worth the issue, but it should be easy to fix._
The comment for radiation decay states the wrong time unit, hugely overestimating the time it takes for radiation to decay. 

#### Steps to reproduce:
1. Look at source code and see that radiation resets at (line 81 of RadiationManager.java) MIN_MAGNITUDE = 0.00001.
2. Look at sourceDecayRate, 0.9995D (line 198 of GeneralConfig.java)
3. Look at comment describing decay "Radiation sources are multiplied by this constant roughly once per second to represent their emission decay. At the default rate, it takes roughly 10 days to remove a 1,000 Sv/h (crazy high) source."
4. Calculate log(0.00001/1000)/log(0.9995) = 36832 seconds
5. Realize the following:
a day is 864000 seconds long (10 days is 8640000)
a minecraft day is 20 minutes long thus 72000 seconds (10 days is 720000)
an hour is 3600 seconds long (10 hours is 36000)
8. Realize that 36832 ≈ 36000 thus comment should talk about 10 realtime hours or 30 minecraft days or value should be changed to (0.00001/1000)^(1/8640000) ≈  0.999998.

TLDR; 10 days not correct, instead 10 hours or 30 minecraft days.

#### Version (make sure you are on the latest version before reporting):
Forge: forge-1.16.5-36.0.45
Mekanism: Mekanism-1.16.5-10.0.21.448
Other relevant version: direwolf20 1.16 modpack

#### Math screenshots:
Calculation for decay
![image](https://user-images.githubusercontent.com/25267070/115150054-0a4fa700-a067-11eb-92ed-9b65c8759838.png)

Reverse calculation for decay
![image](https://user-images.githubusercontent.com/25267070/115149824-1b4be880-a066-11eb-9b77-478a3f9a0dc5.png)

Different decay value for 10 day decay of a 1,000 Sv/h (crazy high) source
![image](https://user-images.githubusercontent.com/25267070/115149965-aaf19700-a066-11eb-8784-dca94f17ec58.png)
